### ISSUE TITLE
Implement lazy loading on all sections.

### ISSUE THREAD

**Issue by Ace-Bansal** [opencodeiiita/Opencode-Collaborative-19#87]
This issue couldn't be solved in the last edition of OpenCode. Let's see if someone can solve it this time around!

**Comment by lo0oper** [comment]
just mentioning the comment to claim it is enough for claiming it right?
i want to claim this

**Comment by lo0oper** [comment]
@opencodebot claim

**Comment by lo0oper** [comment]
@opencodebot claim

**Comment by ananya-agrawal** [comment]
@lo0oper issue is assigned to you

**Comment by zerefwayne** [comment]
@lo0oper Check your mail and accept the invitation as well.

**Comment by lo0oper** [comment]
checked , got it . do i need to do anything else to claim the issue.

**Comment by Ace-Bansal** [comment]
Nope. This issue is yours.

**Comment by lo0oper** [comment]
thankx...
i will get this in the assigned issues , right?

**Comment by lo0oper** [comment]
thankx got it....

**Comment by lo0oper** [comment]
okay...
can you tell me where to start with it ?

**Comment by vishnuk7** [comment]
@opencodebot claim

**Comment by arpitmisraw** [comment]
"Couldn't be solved last time" 😭😭😭

**Comment by anupm12** [comment]
@opencodebot claim

**Comment by sourabh1031** [comment]
@opencodebot claim

**Comment by sourabh1031** [comment]
Can i work on this ?

**Comment by sourabh1031** [comment]
@anupm12 i think you have been assigned three issues . Can I try on this

**Comment by anupm12** [comment]
> @anupm12 i think you have been assigned three issues . Can I try on this

@sourabh1031 
No. I have 2 issues assigned among those, one I have made a pr already
Let me try it once and if I'm unable to do it then you can take it.
@Jigar3 @Ace-Bansal @ananya-agrawal @zerefwayne

**Comment by sourabh1031** [comment]
@anupm12 you have been assigned three issues. Can i claim it now

**Comment by vishnuk7** [comment]
@Jigar3 @Ace-Bansal Can i work on this issue?

**Comment by ananya-agrawal** [comment]
@anupm12 so are you working on this?

**Comment by anupm12** [comment]
@ananya-agrawal yes I will
@opencodebot claim

**Comment by ananya-agrawal** [comment]
@anupm12 don't take too many issues in hand man :stuck_out_tongue:

**Comment by anupm12** [comment]
@ananya-agrawal I'm almost done with the other two na 😅

**Comment by ananya-agrawal** [comment]
Haha okay

**Comment by anupm12** [comment]
@ananya-agrawal @Ace-Bansal Lazy loading has to be applied to images of all sections right?

**Comment by anupm12** [comment]
@ananya-agrawal I'm making use of lozad.js and when I use _data-src_ property instead of _src_ the images won't load at all. Can you please suggest me some solution to this.
Also, I wanted to know fetching data from JSON will be static fetching or dynamic fetching?

**Comment by anupm12** [comment]
Mentors please help me.

**Comment by anupm12** [comment]
@opencodebot unclaim

**Comment by vishnuk7** [comment]
@opencodebot claim
